src/components/table/table-lite.js

```diff
@@ -0,0 +1,39 @@
+import Vue from '../../utils/vue'
+
+// Mixins
+import idMixin from '../../mixins/id'
+import normalizeSlotMixin from '../../mixins/normalize-slot'
+
+// Table helper Mixins
+import itemsMixin from './helpers/mixin-items'
+import captionMixin from './helpers/mixin-caption'
+import colgroupMixin from './helpers/mixin-colgroup'
+import theadMixin from './helpers/mixin-thead'
+import tfootMixin from './helpers/mixin-tfoot'
+import tbodyMixin from './helpers/mixin-tbody'
+
+// Main table renderer mixin
+import tableRendererMixin from './helpers/mixin-table-renderer'
+
+// b-table-lite component definition
+// @vue/component
+export default Vue.extend({
+  name: 'BTableLite',
+  // Order of mixins is important!
+  // They are merged from left to fight, followed by this component.
```

Comment:
`left to fight` => `top to bottom` or `first to last`

Reply:
Top to bottom because of prettier ;-)